src/Controllers/gdpremail.php

```diff
@@ -0,0 +1,68 @@
+#!/usr/local/bin/php -q
+<?php
+
+use \MyRadio\Config;
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadio\AuthUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+use \MyRadio\MyRadio\MyRadioDefaultAuthenticator;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once __DIR__.'/root_cli.php';
+
+$db = Database::getInstance();
+
+$warning_email = <<<EOT
```

Comment:
It could but with the context of it only being sent out by a manually called php script I didn't really see the point of making it easily configurable.